assets/xml/objects/object_hakach_objects.xml

```diff
@@ -0,0 +1,28 @@
+<Root>
+    <File Name="object_hakach_objects" Segment="6">
+        <DList Name="gBotwHoleTrap1Dlist" Offset="0x01B0"/>
+        <DList Name="gBotwHoleTrap2Dlist" Offset="0x03F0"/>
+        <DList Name="gBotwTumbLidDlist" Offset="0x06B0"/>
+        <DList Name="gBotwBombSpotDlist" Offset="0x0A10"/>
+        <DList Name="gBotwFakeWallsAndFloorsDlist" Offset="0x1060"/>
+        <DList Name="gBotwBloodSplatterDlist" Offset="0x1250"/>
+        <DList Name="gBotwThreeFakeFloorsDlist" Offset="0x1920"/>
+        <DList Name="gBotwWaterRingDlist" Offset="0x1E50"/>
+        <DList Name="gBotwWaterFallTex" Offset="0x2010"/>
+
+        <Collision Name="gBotwTumbLidCol" Offset="0x0870"/>
+        <Collision Name="gBotwBombSpotCol" Offset="0x0C2C"/>
+        <Collision Name="gBotwCol1" Offset="0x1830"/>
+        <Collision Name="gBotwCol2" Offset="0x1AB8"/>
+
+        <Texture Name="gBotwWaterTex1" OutName="water_1" Format="ia16" Width="32" Height="32" Offset="0x028F0"/>
+        <Texture Name="gBotwWaterTex2" OutName="water_2" Format="rgb5a1" Width="32" Height="32" Offset="0x020F0"/>
+        <Texture Name="gBotw_30F0Tex" Format="rgb5a1" Width="32" Height="32" Offset="0x30F0"/>
+        <Texture Name="gBotwBloodSplatterTex" OutName="blood_splatter" Format="ia16" Width="32" Height="32" Offset="0x38F0"/>
+        <Texture Name="gBotw_40F0Tex" Format="rgb5a1" Width="32" Height="32" Offset="0x40F0"/>
+        <Texture Name="gBotw_48F0Tex" Format="rgb5a1" Width="32" Height="32" Offset="0x48F0"/>
+        <Texture Name="gBotw_50F0Tex" Format="rgb5a1" Width="32" Height="32" Offset="0x50F0"/>
+        <Texture Name="gBotw_58F0Tex" Format="rgb5a1" Width="32" Height="32" Offset="0x58F0"/>
+        <Texture Name="gBotw_60F0Tex" Format="i4" Width="32" Height="32" Offset="0x60F0"/>
```

Comment:
Could you try to describe those textures too? They look like they would be wall textures. Maybe their dlists could give you a hint what they are suppose to be.